aten/src/ATen/test/complex_test.cpp

```diff
@@ -0,0 +1,28 @@
+#include <gtest/gtest.h>
+#include "c10/util/Complex.h"
+
+template<typename T, typename int_t>
+static void TestBinaryOpsForIntType(T real, T img, int_t num) {
+  std::complex<T> c(real, img);
+  ASSERT_EQ(c + num, std::complex<T>(real + num, img + num));
+  ASSERT_EQ(num + c, std::complex<T>(num + real, num + img));
+  ASSERT_EQ(c - num, std::complex<T>(real - num, num - num));
+  ASSERT_EQ(num - c, std::complex<T>(num - real, num - img));
+  ASSERT_EQ(c * num, std::complex<T>(real * num, img * num));
+  ASSERT_EQ(num * c, std::complex<T>(num * real, num * img));
+  ASSERT_EQ(c / num, std::complex<T>(real / num, img / num));
+  ASSERT_EQ(num / c, std::complex<T>(num / real, num / img));
+}
+
+template<typename T>
+static void TestBinaryOpsForAllIntTypes(T real, T img, int8_t i) {
+  TestBinaryOpsForIntType<T, int8_t>(real, img, i, op);
+  TestBinaryOpsForIntType<T, int16_t>(real, img, i, op);
+  TestBinaryOpsForIntType<T, int32_t>(real, img, i, op);
+  TestBinaryOpsForIntType<T, int64_t>(real, img, i, op);
```

Comment:
Where is the `op` variable coming from? ovrsource tests started failing on this.

Reply:
Hmm it was probably misadded during modification. I'll remove them. Thanks for the report.